packages/ui/src/theme/tokens/components/alert.ts

```diff
@@ -1,42 +1,38 @@
-import {
-  AlignItemsValue,
-  BackgroundColorValue,
-  ColorValue,
-  DesignToken,
-  FontSizeValue,
-  FontWeightValue,
-  JustifyContentValue,
-  SpaceValue,
-} from '../types/designToken';
+import { DesignTokenProperties } from '../types/designToken';
 
-interface AlertVariationTokens {
-  color: DesignToken<ColorValue>;
-  backgroundColor: DesignToken<BackgroundColorValue>;
-}
+type AlertVariationTokens<OutputType> = DesignTokenProperties<
+  'backgroundColor' | 'color',
+  OutputType
+>;
 
-interface AlertIconTokens {
-  size: DesignToken<FontSizeValue>;
-}
+type AlertIconTokens<OutputType> = DesignTokenProperties<'size', OutputType>;
```

Comment:
`DesignTokenProperties` ideally speeds up development and gives us type safety:

https://user-images.githubusercontent.com/15002885/200149487-e0951548-7eba-464b-8c32-9ef32775faa4.mov